src/xdocs/config_misc.xml

```diff
@@ -895,10 +895,10 @@ String unitAbbrev = "\u03bc\u03bc\u03bc";
     </section>
 
     <section name="FinalParameters">
+      <p>Since Checkstyle 3.0</p>
       <subsection name="Description" id="FinalParameters_Description">
-        <p>Since Checkstyle 3.0</p>
         <p>
-          Check that parameters for methods, constructors, and catch blocks are
+          Check that parameters for methods, constructors, and catch, foreach blocks are
```

Comment:
`and catch, foreach` => `catch, and foreach`
Should this be for-each or just foreach?

Reply:
done. For other checks we use `for-each`